pkg/store/errors_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
2020

Reply:
Oh funny. Want me to update other headers in a follow up?

Reply:
:) no need to update... might be best to remove the year(s) per the way containerd/containerd is doing it at some point in time.